src/applications/disability-benefits/526EZ/config/form.js

```diff
@@ -229,18 +230,83 @@ const formConfig = {
           title: 'Special Circumstances',
           path: 'special-circumstances',
           uiSchema: {
-            'ui:title': 'Homelessness',
-            'view:homelessness': {
-              'ui:title': 'Are you homeless or at risk of becoming homeless?',
-              'ui:widget': 'yesNo',
-            },
+            veteran: {
+              'ui:title': 'Homelessness',
+              homelessness: {
+                isHomeless: {
+                  'ui:title': 'Are you homeless or at risk of becoming homeless?',
+                  'ui:widget': 'yesNo',
+                },
+                pointOfContact: {
+                  'ui:description': 'Please provide the name and number of a person we should call if we need to get in touch with you.',
+                  'ui:options': {
+                    expandUnder: 'isHomeless',
+                    expandUnderCondition: (homelessOrAtRisk) => (homelessOrAtRisk === true)
```

Comment:
Is `expandUnderCondition` necessary if the condition is true? It might not be needed here.

Reply:
You're right! That's nice.